modules/nextflow/src/main/groovy/nextflow/scm/AssetManager.groovy

```diff
@@ -589,6 +591,7 @@ class AssetManager {
             clone
                 .setURI(cloneURL)
                 .setDirectory(localPath)
+                .setCloneSubmodules(true)
```

Comment:
Not sure about this. Modules are updates separately, see 

https://github.com/nextflow-io/nextflow/blob/a34b1d2a3786e04c55dc94906394d4079b40c911/modules/nextflow/src/main/groovy/nextflow/scm/AssetManager.groovy#L952-L969

Reply:
OK. But without this line, I couldn't pull the grand-daughter module, which I need. The daughter modules were OK, though.

Reply:
Not sure how to help. We need to keep the clone of submodules as opt-in feature. My understanding is that your proposal makes it the default. My suggestion is to avoid a nested substructure in your submodules.